docs-src/references/sdk-metrics.md

```diff
@@ -7,10 +7,12 @@ tags:
   - reference
 ---
 
-> For Cluster metrics, see [Cluster ▶️ Production deployment ▶️ Scaling and Metrics](/server/production-deployment/#scaling-and-metrics).
```

Comment:
Do we have all this info elsewhere, or should I link to this in addition? 

https://docs.temporal.io/server/production-deployment#scaling-and-metrics

Reply:
Keep that link for now.
But moving this into and expanding it in the Deployment guide is a top priority for us these next 2 Quarters.

Reply:
Actually on closer inspection, much of this is captured in the Cluster metrics reference - 
And if it is not - then it should be captured there - or in the Cluster deployment guide
We want to disperse the info in that file into the appropriate places in the new guides.